build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+read -p "Build dataplicity agent from PyPi? " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
```

Comment:
Usually better to put "" around variables used like this, in case they are empty.  Limited scope for issue here as this is a build script but still no harm in it.